tests/sentry/ratelimits/utils/test_enforce_rate_limit.py

```diff
@@ -0,0 +1,58 @@
+from django.conf.urls import url
+from django.test import override_settings
+from freezegun import freeze_time
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.testutils import APITestCase
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+
+class RateLimitEnforcedEndpoint(Endpoint):
```

Comment:
This might be overkill but:
```py
class RateLimitTestEndpoint(Endpoint):
    permission_classes = (AllowAny,)
    rate_limits = {"GET": {RateLimitCategory.IP: RateLimit(1, 100)}}

    def get(self, request):
        return Response({"ok": True})

class RateLimitEnforcedEndpoint(RateLimitTestEndpoint):
    enforce_rate_limit = True

class RateLimitUnenforcedEndpoint(RateLimitTestEndpoint):
    enforce_rate_limit = False
```
might make it easier to highlight the differences between the two endpoints.